content/graphing/infrastructure/hostmap.md

```diff
@@ -12,114 +12,118 @@ further_reading:
-* upgrade_required : the Datadog Agent requires an upgrade
+### Shapes and colors
 
-### Group hosts by tags
+By default the color of each host is set to represent the percentage of CPU usage on that host, where the color ranges from green (0% utilized) to red (100% utilized). You can select different metrics from the `Color by` selector.  
 
-`Group hosts by tags` spatially arranges your hosts into clusters, or groups. Any host in a group shares the tag or tags you group by.  
-A simple example is grouping your hosts by AWS availability zone. If you add a second grouping tag, such as instance type, then the hosts are further subdivided into groups, first by availability zone and then by instance type, as seen below.
+Host Maps can also communicate an additional, optional metric with the size of the hexagon: use the `Size by` selector. 
 
-{{< img src="graphing/infrastructure/hostmap/hostmappart2image2.png" alt="Datadog Host Maps AZ Instance Groups" responsive="true" >}}
+In the screenshot below, the size of the hexagons is the 15 minute average load, normalized so that machines' workloads can be compared even if they have different numbers of cores.
 
-### Zoom in
+{{< img src="graphing/infrastructure/hostmap/hostmappart2image4.png" alt="Datadog Host Maps Using Color And Size" responsive="true" style="width:80%;">}}
 
-When you've identified a host that you want to investigate, click it for details. It zooms in and see up to six integrations reporting metrics from that host. (If there are more than six integrations, they are listed under the "Apps" header in the host's detail pane, as in the screenshot below).  
-Click the name of an integration  to get a condensed dashboard of metrics for that integration. In the screenshot below, we have clicked "system" to get system metrics such as CPU usage, memory usage, disk latency, etc.
+**Note**: The "% CPU utilized" metric uses the most reliable and up-to-date measurement of CPU utilization, whether it is being reported by the Datadog Agent, or directly by AWS or vSphere.
 
-{{< img src="graphing/infrastructure/hostmap/blog-host-maps-01.png" alt="Datadog Host Maps Zoom In" responsive="true" style="width:75%;" >}}
+### Display hosts on the Host Map that don't have an Agent installed
 
-### Shapes and colors
+By default, the Host Map only shows hosts that are reporting certain metrics, which can then be used to set a color or size for the individual hexagon within the grid.
 
-By default the color of each host (hexagon) is set to represent the percentage of CPU usage on each host, where the color ranges from green (0% utilized) to red (100% utilized). You can select different metrics from the `Color by` selector.  
+Some hosts, like those coming in from Google App Engine, do not pull in these metrics without an Agent running. These hosts can still appear within the Host Map by selecting the "gear" icon on the top-right of the map and enabling "Show hosts with no metrics" in the Host Map settings:
```

Comment:
```suggestion
If a host is not reporting the selected metric, it can still appear within the Host Map by selecting the "gear" icon on the top-right of the map and enabling "Show hosts with no metrics" in the Host Map settings:
```